pkg/kubelet/types/constants.go

```diff
@@ -20,3 +20,13 @@ const (
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
 )
+
+// DefaultSysctlWhitelist returns the default whitelist of sysctls and sysctl patterns (ending in *).
+func DefaultSysctlWhitelist() []string {
+	return []string{
+		"kernel.shm*",
```

Comment:
For shmmax and shmall I agree. For shmmni (defaults to 4096 in today's kernels):

- Oracle: 4096 minimal 
- IBM DB2: 256*GB-of-RAM > 4096 for more than 16 GB of RAM

WDYT?